### PR TITLE
Cli tab: vertical flex fix and removing scroll bars

### PR DESCRIPTION
**Pull request by limonspb**
Before PR (when window is small enough):
image

After PR:
image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]

**Review by chmelevskij — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
